tests/aio_tests/notest_plugable_switch_measuring_async.py

```diff
@@ -22,6 +22,11 @@ def test_update_event(fake_switch):
     fake_switch.fire_update_event()
     fake_handler.method.assert_called()
 
+def test_remove_event(fake_switch):
```

Comment:
Could you move this to another file please?
all "notest_*" files will not be tested. I've implemented a fake cloud server for more "realistic" tests instead of mocking. Websockets are currently not in the cloud, so I'm fine with mocking this, but please make sure, that the tests are running

Reply:
I removed this test. because same funftionality is alread mock tested for home.